### ISSUE TITLE
No section: u'mri-server-dispatch'

### ISSUE THREAD

**Issue by haoyangz** [Mri-monitoring/Mri-app#10]
I am a new user and was trying to test Mri on a toy example using the config.Template. I got the following error:

```
Traceback (most recent call last):
  File "MriApp.py", line 25, in <module>
    main()
  File "MriApp.py", line 21, in main
    client = MriCaffe(arg.config, arg.solver_override)
  File "build/bdist.linux-x86_64/egg/mriapp/MriCaffe.py", line 37, in __init__
    verify_config(config_file)
  File "build/bdist.linux-x86_64/egg/mriapp/utilities/verify_config.py", line 36, in verify_config
  File "/home/zeng/miniconda/lib/python2.7/ConfigParser.py", line 607, in get
    raise NoSectionError(section)
ConfigParser.NoSectionError: No section: u'mri-server-dispatch'
```

I simply ran 

```
python MriApp.py
```

My config file is basically the same as the template, with the server part deleted:

```
; ---=== Dispatch Settings ===---
; Dispatches events to be plotted locally in Matplotlib
[matplotlib-dispatch]
save_img_folder = ../saveimages                 ; Location to save matplotlib outputs
show_windows = False                                    ; Optional, set to True to show plots during training

; ---=== Retrieve Settings ===---
; Retrieve events from a local folder
[local-retrieve]
task_list = /media/bigdrive/testmodel/mri/task/generated_tasks.txt          ; Local list of tasks, simply a list of folders containing tasks

; ---=== Client Settings ===---
[mri-client]
log_location = ../mri.log                               ; Log save location
debug = True                                            ; Enable for verbose logging and stdout
caffe_root = /media/bigdrive/caffe                           ; Root caffe directory. Caffe must be built!
caffe_bin = ./build/tools/caffe                         ; Path to caffe binary from root caffe folder
solver_type = Caffe                                     ; Currently only Caffe is possible

; Dispatch and Retrieve modules to use
dispatch = mri-server-dispatch                          ; Run with the server dispatch
retrieve = local-retrieve                               ; Run with the local retriever
```

If I delete the last two lines, I will then got :

```
Traceback (most recent call last):
  File "MriApp.py", line 25, in <module>
    main()
  File "MriApp.py", line 21, in main
    client = MriCaffe(arg.config, arg.solver_override)
  File "build/bdist.linux-x86_64/egg/mriapp/MriCaffe.py", line 37, in __init__
    verify_config(config_file)
  File "build/bdist.linux-x86_64/egg/mriapp/utilities/verify_config.py", line 30, in verify_config
  File "/home/zeng/miniconda/lib/python2.7/ConfigParser.py", line 618, in get
    raise NoOptionError(option, section)
ConfigParser.NoOptionError: No option u'dispatch' in section: u'mri-client'
```

**Comment by haoyangz** [comment]
After reading the source code of _varify_config.py_, I realize that if I use local dispatch, I should change the value of `dispatch`  to matplotlib-dispatch . This is not clear for first-time users, and some comments are needed.

**Comment by nharada1** [comment]
Hi,

Thanks for the heads up! Just to be clear, what happened was you deleted the mri-server-dispatch section and then it failed? I agree this is unclear and will change the example.

**Comment by haoyangz** [comment]
Yes. I guess you can maybe split the config.Tempalte into two tempaltes, one for matplotlib and one for the server. Alternatively, if you parser supports comments, maybe put the code for matplotlib in the template, and write the code for server in the comment to demonstrate the alternative setting?

**Comment by haoyangz** [comment]
fixed in commit 5753f169e14798384d4b7b3b33853d6ab7433b0e
